src/Option.ts

```diff
@@ -204,6 +225,27 @@ export class Some<A>
   toString() {
     return `some(${toString(this.value)})`
   }
+  contains(setoid: Setoid<A>, a: A): boolean {
```

Comment:
Good point, I've tried to mirror the signature of the `equals` method, but an optional `setoid` would work as well of course.  Let me know what you prefer, and I'll update if needed.